src/server/project.ts

```diff
@@ -142,6 +154,35 @@ namespace ts.server {
             return this.cachedUnresolvedImportsPerFile;
         }
 
+        public static resolveModule(moduleName: string, initialDir: string, host: ServerHost, log: (message: string) => void): {} {

```

Comment:
@weswigham I don't really see a point - is anyone plausibly going to write a plugin that works under a host that's not node?